.github/workflows/nightly.yml

```diff
@@ -72,6 +72,21 @@ jobs:
           name: vector-${{ env.VECTOR_VERSION }}-1.aarch64.rpm
           path: "./target/artifacts/vector-${{ env.VECTOR_VERSION }}-1.aarch64.rpm"
 
+  build-aarch64-unknown-linux-gnu-packages:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - run: make ci-sweep
+      - run: echo VECTOR_VERSION=$(make version) >> $GITHUB_ENV
+      - env:
+          DOCKER_PRIVILEGED: "true"
+          PASS_FEATURES: "default-cmake"
```

Comment:
Should this just be `default` to match `build-x86_64-unknown-linux-gnu-packages:`?